README.md

```diff
@@ -1,140 +1,126 @@
+run the following commands:
 
-To create a new project using this template, in the folder where you want your project to be located, run the following
-code in your terminal:
-
-```shell script
+```shell
 cookiecutter https://github.com/ukgovdatascience/govcookiecutter
 ```
 
-Then following the prompts in your terminal to create the project structure.
+Follow the prompts, and that's it — you've created your project structure! Post-creation, there are a few
+[mandatory changes](#changes-to-make-post-creation) to make, as well as a some
+[optional changes](#changes-to-consider-post-creation) you should also consider making.
+
+Otherwise, that's it — happy coding! 🎉
 
-## Installing pre-commit hooks
+### Requirements
 
-This repo uses the Python package [`pre-commit`](https://pre-commit.com) to manage pre-commit hooks. Pre-commit hooks
-are actions which are run automatically, typically on each commit, to perform some common set of tasks. For example, a
-pre-commit hook might be used to run any code linting automatically, providing any warnings before code is committed,
-ensuring that all of our code adheres to a certain quality standard.
+> ℹ️ Contributors have some additional requirements! Check out the [contributing guidelines][contributing] for further
+> details.
 
-For this repo, we are using `pre-commit` for a number of purposes:
-- Checking for any secrets being committed accidentally;
-- Checking for any large files (over 5MB) being committed; and
-- Cleaning Jupyter notebooks, which means removing all outputs and execution counts.
+To get started your system should meet the following requirements:
 
-We have configured `pre-commit` to run automatically on _every commit_. By running on each commit, we ensure
-that `pre-commit` will be able to detect all contraventions and keep our repo in a healthy state.
+1. Unix-based system (macOS, Linux, ...)
+2. Python 3.5+ installed
+3. The [`cookiecutter` package installed](#installing-cookiecutter)
 
-In order for `pre-commit` to run, action is needed to configure it on your system.
-- [Install](#installing-python-packages) the `pre-commit` package into your Python environment from
-`requirements-dev.txt`; and
-- Run `pre-commit install` to set-up `pre-commit` to run when code is _committed_.
+#### Installing cookiecutter
 
-### Setting up a baseline for the `detect-secrets` hook (if one doesn't already exist)
+There are many ways to install the [`cookiecutter`][cookiecutter] package. **Our recommendation is to install it at the
+system or user level**, rather than as a Python package via `pip` or `conda`. This ensures it is isolated from the rest
+of your system, and always available.
 
-The `detect-secrets` hook requires that you generate a baseline file if one is not already present within the root
-directory. This is done via running the following at the root of the repo:
+For macOS, install `cookiecutter` via [Homebrew][homebrew]:
```

Comment:
I think we should be explicit and say you run the following in your terminal.

Reply:
Changed in commit [`c715927`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/c71592757be0e1dd4dae0ce7c4bdfd63b56f1134). Marking as **Resolved**.